central/tlsconfig/tlsconfig.go

```diff
@@ -183,3 +215,32 @@ func validForAllDNSNames(cert *x509.Certificate, dnsNames ...string) bool {
 	}
 	return true
 }
+
+var allowedAdditionalCAExtensions = map[string]bool{
+	".crt": true,
+	".pem": true,
```

Comment:
Could the file extension be ignored altogether? 
Just scanning all files, and if a file can't be decoded, log a warning message instead?

Reply:
That would be ideal yes. I would like to do this in another PR, because I would need the symlink logic in #5575 . Perhaps I can add a ticket to the epic, the basically remove any sort of extension restriction for the additional-cas, and address it there. What do you think?

Reply:
I think it is okay. 
To me it sounded like ignoring the file extension reduces the needed logic for error reporting and logging, instead it is reduced to writing encoding errors to the logger.